api/tests/test_meals.py

```diff
@@ -0,0 +1,76 @@
+from app import app
+import unittest
+import json
+
+
+
+
+class TestMeal(unittest.TestCase):
+
+    def setUp(self):
+        app.testing = True
+
+        self.app = app.test_client()
+
+        self.data = {
+                        "id":1,
+                        "meal_name":"lunch"
+        				 }
+       
+
+    def test_get_and_create_meals(self):
```

Comment:
same here, test get meals separately and create meals separately

Reply:
noted,I am working on it